sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.1.0-beta.1 (Unreleased)
+## 12.1.0 (2021-07-08)
 
 ### New Features
```

Comment:
Update according to the change log guidance here https://azure.github.io/azure-sdk/policies_releases.html#change-logs

i.e. ### Features Added, ### Breaking Changes, ### Bugs Fixed